tests/util.h

```diff
@@ -66,6 +66,10 @@ static void serializePacket(T &packet, const QByteArray &xml)
     auto processedXml = xml;
     processedXml.replace(u'\'', u'"');
 
+    // Remove newlines and needless spaces from raw strings.
+    processedXml = processedXml.simplified();
+    processedXml.replace("> <", "><");
+
```

Comment:
this will also remove whitespace *in* the xml, so e.g. in the body of message, I'm not sure if I want that...

Reply:
the correct way would be to parse and re-serialize both xml representations (or even doing the comparison with QDom or another parsed representation, with more helpful errors), but yeah .. I guess it's okay for now and it will work in most cases